runtime/lua/lsp/json.lua

```diff
@@ -0,0 +1,13 @@
+
+local json = {}
+
+-- TODO: Use the FFI module to decode and encode items, since that should be faster
```

Comment:
If you still plan to do this, please wait until a separate PR. It will probably involve changes to the build process, dependencies, etc. And I suspect the performance won't actually matter.

Reply:
Before marking ready for review, I'll try and make sure this is true, but I plan on having it be: plain-old `TODO: ...` comments are things that are out of the scope of this PR but could be cool/nice. I'll try and change the TODOs I want to get to into `TODO(tjdevries): ...`